app_dart/lib/src/service/luci_build_service.dart

```diff
@@ -86,7 +94,24 @@ class LuciBuildService {
     return getBuilds(slug, sha, builderName, 'try', tags);
   }
 
-  /// Returns an Iterable of try Buildbucket [Build]s for a given [PullRequest].
+  Future<Iterable<bbv2.Build>> getTryBuildsV2(
+    github.RepositorySlug slug,
+    String sha,
+    String? builderName,
+  ) async {
+    final List<bbv2.StringPair> tags = [
+      bbv2.StringPair(key: 'buildset', value: 'sha/git/$sha'),
+      bbv2.StringPair(key: 'user_agent', value: 'flutter-cocoon'),
+    ];
+    return getBuildsV2(
+      slug,
+      sha,
+      builderName,
+      'try',
+      tags,
+    );
+  }
```

Comment:
Do we still need this considering the logic is also in `luci_build_service_v2.dart`?

Reply:
It is for now to allow this service to publish to the new version v2 rather than having a separate dependency. Simplicity was my aim here. It will be removed once we are publishing strictly to v2.